src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,6 +15,13 @@
 
 error_reporting(-1);
 
+$passthru_or_fail = function ($command) {
```

Comment:
we use camel case :)
but `$passthru` would be fine as a name to me

Reply:
Will change, but prefer to keep the `orFail` to make clear there's a difference to plain `passthru()`